public/static/docs/user-guide/privacy.md

```diff
@@ -42,8 +42,16 @@ section of this User Guide.
 
 ## Scopes
 
+`userinfo.email` scope (view your email address), `userinfo.profile` scope (see
+your personal info, including any personal info you've made publicly available),
+and `openid` scope (authenticate using OpenID Connect) are required to be used
+by Google Auth. DVC authorizes users via
+[Google's server](https://accounts.google.com/o/oauth2/auth) and doesn't access
```

Comment:
Hi! This link doesn't work like this. Should we just display it instead like `` `https://accounts.google.com/o/oauth2/auth` `` ?

Reply:
Hey @jorgeorpinel , the idea here was to mention explicitly that server belongs to Google and link was added just as a prove. Also link might change with time.

Reply:
Also not sure why it doesn't work? I can click at "Google's server" and the URL is opened in new tab ( error in new tab is displayed, but it is fine )

Reply:
I think @jorgeorpinel refers to the fact that it returns 404 and looks like broken bc of that. Also trigger our CI scripts that detect broken/dead links. I've added it to the exclusion list for now. Still, not clear if it makes sense to keep it this way, even though I understand the intention.

Reply:
Yep, we need it in the link exclusion list anyway, but it's best to display the link in the markdown like a `` `quote` `` instead of a clickable link, I think.

Reply:
And yes, I meant it's not a link to any useful website. It doesn't work like that = it's not meant to be navigated, just the base URL to begin a OAuth authentication (but needs a special token in the URL query)

Reply:
@jorgeorpinel got it, thanks, should I propose the changes?

Reply:
@jorgeorpinel let's fix it as part of the regular updates flow.

Reply:
Done in 45e93dc6. I reorganized and reworded some paragraphs along the way so please review the draft PR (#987).

Reply:
p.s. I ay have some more Qs, will add them there.